dojo/templates/dojo/view_finding.html

```diff
@@ -440,6 +440,29 @@ <h3 class="pull-left finding-title">
         </div>
     </div>
 
+    {% if finding.epss_score or finding.epss_percentile %}
+    <div class="panel panel-default">
+        <table id="additional_finding_references" class="table-striped table table-condensed table-hover centered">
```

Comment:
Rather than having this as a separate table beneath the main entry, could we just add a "EPSS Score / Percentile" column to the main entry above? It could be a single slash-separated value (e.g. `41.1% / 11.7%`). The logic for display would still be the same, "N.A." values could still work there.

Reply:
I like what @dogboat proposes. The pages are already getting pretty long to adding EPSS without making the page significantly longer is a great choice.

Thanks dogboat for these great suggestions - much appreciated :tada:

Reply:
Thanks for the input! I will try this as well in the findings list page.